package.json

```diff
@@ -99,5 +98,8 @@
   },
   "homepage": "https://stellar.org",
   "license": "MIT",
-  "prettier": "@stellar/prettier-config"
+  "prettier": "@stellar/prettier-config",
+  "devDependencies": {
+    "prettier": "prettier/prettier"
```

Comment:
I've started avoiding `devDependencies` on apps, because we have to install deps + devDeps anyway for a production build, so it doesn't seem like a useful distinction. devDeps useful for libraries so they don't put more on disk than what's needed

Reply:
Oh also, looks like this is pulling from github? Do we need to avoid the npm version?

Reply:
It looks like our `.prettierrc.js` is being ignored. Its `proseWrap` is set to "never"`, but still wraps it. Investigating it..

Reply:
@vcarl, I figured it out. Making a new PR and will add details. Closing this.